packages/components/src/internal/util/utils.ts

```diff
@@ -470,3 +470,88 @@ export function isNonNegativeInteger(value: number | string): boolean {
 export function isNonNegativeFloat(value: number | string): boolean {
     return isFloat(value) && Number(value) >= 0;
 }
+
+function getFileExtensionType(value: string): string {
+    const parts = value.split('.');
+    return parts[parts.length - 1].toLowerCase();
+}
+
+export function isImage(value): boolean {
+    const validImageExtensions = ['jpg', 'jpeg', 'bmp', 'gif', 'ico', 'png', 'svg', 'tif'];
+    const extensionType = getFileExtensionType(value);
+    return validImageExtensions.indexOf(extensionType) > -1;
+}
+
+export function downloadAttachment(href: string, openInTab?: boolean, fileName?: string): void {
+    if (openInTab) {
+        window.open(href, '_blank');
+    } else {
+        const link = document.createElement('a');
+        link.href = href;
+        link.download = fileName;
+        link.click();
```

Comment:
That's kind of a crafty way to do this. :-)

Reply:
I can't take credit. This was ported from ELN.